assets/translations/ar.json

```
@@ -1,4 +1,14 @@
 {
+  "login": "تسجيل الدخول",
```

Biome (2.1.2)

[error] The key login was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)